dali/ft/filecopy.cpp

```diff
@@ -1945,7 +1981,7 @@ void FileSprayer::cloneHeaderFooter(unsigned idx, bool isHeader)
     if (isHeader)
         next.inputOffset = curSrc.headerSize - curSrc.xmlHeaderLength;
     else
-        next.inputOffset = curSrc.headerSize + curSrc.size;
+        next.inputOffset = curSrc.size + curSrc.xmlHeaderLength;
```

Comment:
I'm a bit confused by this change.. may need a comment..
why in isHeader=false (so footer?) .. is it adding xmlHeaderLength here?